src/utils/rest-api.js

```diff
@@ -577,7 +577,7 @@ export function fetchEquipmentsInfos(
     studyUuid,
     nodeUuid,
     searchTerm,
-    usesName,
+    getParameter,
```

Comment:
it's a string here not a function, then name it `useNameParameterKey` ?

Reply:
it is a function it will be executed when the `urlSearchParams.append('fieldSelector', getUseNameParameterKey)` is called